operators/src/error.rs

```diff
@@ -314,6 +314,21 @@ pub enum Error {
     QueryCanceled,
 
     AbortTriggerAlreadyUsed,
+
+    SubPathMustNotEscapeBasePath {
+        base: PathBuf,
+        sub_path: PathBuf,
+    },
+
+    InvalidDataProviderConfig,
+    InvalidMachineLearningConfig,
+    NotAvailable,
```

Comment:
I meant `MachineLearningModelNotAvailable` or something like that

Reply:
Fingers crossed, that it is correct now https://github.com/geo-engine/geoengine/pull/639/commits/123c769ebd9ef44cce0d569bfaa46c1471ea7885.